examples/basic/rec-auto-stereo.ck

```diff
@@ -14,6 +14,8 @@ dac => WvOut2 w => blackhole;
 
 // this is the output file name
 "special:auto" => w.wavFilename;
```

Comment:
would be sick (in another PR / round of feature updates) to be able to replace "special:auto" with a ChucK-native solution (basically datetime -> string support I think). Datetime is a massive yak shave but I think languages like Python got it pretty well down!

Reply:
a good yak to aspire to!